vuepress/docs/next/tutorials/devops/installation/kubernetes/kubernetes-install.md

```diff
@@ -0,0 +1,47 @@
+# Install Entando on Kubernetes
+
+## Prerequisites
+
+### Infrastructure Requirements
+
+To install Entando directly on a Kubernetes (K8s) cluster, these are the minimum infrastructure requirements:
+
+- A working DNS server configured to resolve internal and external names. If you choose to configure K8s with a global wildcard domain name for the entire cluster, be sure it points to the public IP address of the external Load Balancer (LB) of the worker nodes.
+- A default [Storage Class](https://kubernetes.io/docs/concepts/storage/storage-classes/) is needed for the Entando operator to create the correct Persistent Volume Claims (PVC) for each pod.
+- Nginx [Ingress-Controller](https://kubernetes.github.io/ingress-nginx/deploy/) is needed to expose the Entando applications.
+
+### Minimum Hardware Requirements
+
+The minimum hardware requirements depends on how you intend to use Entando 6. The more functionality you need such as a content management system (CMS), the more you will need.
+#### For Entando install only:
+
+- 6 vCPU or virtual central processing unit  (6000 millicores)
```

Comment:
Acronyms should be treated consistently (unabbreviated phrase followed by acronym in parentheses)?

Reply:
it's legit to do it this way too, to use in awkward places like this when it would make two parenthesis right next to each other.

Reply:
totally agree but wanted to make sure that overrode consistency